vminfo_parser/clioutput.py

```diff
@@ -86,110 +87,35 @@ def format_series_output(self: t.Self, counts: pd.Series) -> None:
-            formatted_row = [str(index).ljust(justification)]
-            for col_name, width in col_widths.items():
-                value = str(row[col_name]) if col_name in row.index else ""
-                formatted_row.append(value.ljust(width))
-            formatted_rows.append(" ".join(formatted_row))
-
-        formatted_df_str = "\n".join(formatted_rows)
-
-        return formatted_df_str
-
-    def set_column_width(
-        self: t.Self,
-        dataFame: pd.DataFrame,
-        index_column_padding: int,
-        remaining_column_padding: int,
-        index_column_name: str = None,
-    ) -> dict:
-        """
-        Set the widths for the columns in a DataFrame based on specified parameters.
-        This function returns a dictionary mapping column names to their respective widths for formatting purposes.
-
-        Args:
-            dataFame (pd.DataFrame): The DataFrame for which column widths are to be set.
-            index_column_width (int): The width to be assigned to the index column.
-            remaining_column_widths (int): The width to be assigned to the remaining columns.
-            index_column_name (str, optional): The name of the index column. If provided, it will be assigned the
-                specified width. Defaults to None.
-
-        Returns:
-            dict: A dictionary mapping each column name to its corresponding width.
-
-        Examples:
-            >>> set_column_width(dataFame=my_dataframe, index_column_width=20, remaining_column_widths=10,
-                index_column_name="Environment")
-            {'Environment': 20, 'Column1': max(10, len('Column1')), 'Column2': max(10, len('Column2'))}
-        """
-        if index_column_name:
-            col_widths = {
-                index_column_name: index_column_padding,
-                **{env: max(remaining_column_padding, len(env)) for env in dataFame.columns},
-            }
-        else:
-            col_widths = {env: max(index_column_padding, len(env)) for env in dataFame.columns}
-
-        return col_widths
-
     def print_formatted_disk_space(
         self: t.Self,
-        col_widths: dict,
-        formatted_df_str: str,
+        dataFrame: pd.DataFrame,
         os_filter: t.Optional[str] = None,
-        display_header: bool = True,
-        index_heading_justification: int = 39,
-        other_headings_justification: int = 11,
     ) -> None:
         """
         Print the formatted disk space information to the output.
         This function displays a header and the formatted data, optionally filtered by the operating system.
 
         Args:
-            col_widths (dict): A dictionary containing the widths for each column in the output.
             formatted_df_str (str): A string representation of the formatted disk space data.
             os_filter (Optional[str]): An optional filter to display specific operating system information.
 
         Returns:
             None
         """
```

Comment:
The docstring defines the data coming in as a string, the signature defines a DataFrame

Reply:
fixed